contracts/base-helper.ts

```diff
@@ -0,0 +1,120 @@
+import axios from  "axios";
+import fs from "fs";
+import { SigningCosmWasmClient } from "@cosmjs/cosmwasm-stargate";
+import { GasPrice, calculateFee, StdFee } from "@cosmjs/stargate";
+import { DirectSecp256k1HdWallet, makeCosmoshubPath } from "@cosmjs/proto-signing";
+import { Slip10RawIndex } from "@cosmjs/crypto";
+import path from "path";
+
+/*
+ * This base-helper is here to use with different helper files in contracts
+ *
+ * Usage: npx @cosmjs/cli@^0.26 --init https://raw.githubusercontent.com/CosmWasm/cw-plus/master/contracts/base-helpers.ts
+ *
+ * Create a client:
+ *   const [addr, client] = await useOptions(pebblenetOptions).setup('password');
+ *
+ * Get the mnemonic:
+ *   await useOptions(pebblenetOptions).recoverMnemonic(password);
+ *
+ * If you want to use this code inside an app, you will need several imports from https://github.com/CosmWasm/cosmjs
+*/
+
+interface Options {
+  readonly httpUrl: string
+  readonly networkId: string
+  readonly feeToken: string
+  readonly bech32prefix: string
+  readonly hdPath: readonly Slip10RawIndex[]
+  readonly faucetUrl?: string
+  readonly defaultKeyFile: string,
+  readonly fees: {
+    upload: StdFee,
+    init: StdFee,
+    exec: StdFee
+  }
```

Comment:
It would make sense to just store gasPrice once and then gas limits for each operation. This makes it much easier to go from network to network because only gas price needs to be updated.

`calculateFee` can calculate a `StdFee` at any time from gas price and gas limit.